src/Locations.js

```diff
@@ -203,54 +207,70 @@ export default class Locations extends Component {
                             <Tabs>
                                 <TabList>
                                 {Object.keys(LOCATIONS).map(locationName => (
-                                    <Tab key={locationName.split(' ').join('-')}>{locationName}</Tab>
+                                    <Tab key={locationName[0].split(' ').join('-')}>{locationName}</Tab>
```

Comment:
This should be `locationGroupName`, and you don't need the `[0]` - please test your changes

Reply:
I'm really not sure what was going through my head with this one. My mistake